pallets/staking/src/lib.rs

```diff
@@ -144,18 +147,20 @@ pub mod pallet {
             let mut weight = Weight::from(0u64);
 
             // migrates our storage to pallet version 2.0.6
-            let version: PalletVersion =
-                <Pallet<T>>::storage_version().unwrap_or(<Pallet<T>>::current_version());
+            // let version: PalletVersion =
+            //     <Pallet<T>>::storage_version().unwrap_or(<Pallet<T>>::current_version());
 
-            log::info!(
-                "on_runtime_upgrade>[{:#?}]=> - Current Pallet Version-[{:#?}]",
-                line!(),
-                version,
-            );
+            // log::info!(
+            //     "on_runtime_upgrade>[{:#?}]=> - Current Pallet Version-[{:#?}]",
+            //     line!(),
+            //     version,
+            // );
 
-            if version.major == 2 && version.minor == 0 {
-                weight = migrations::poa_validators_migration::<T>();
-            }
+            // if version.major == 2 && version.minor == 0 {
+            //     weight = migrations::poa_validators_migration::<T>();
+            // }
```

Comment:
Let's discuss whether this should be removed or kept as is on our call?

Reply:
@ETeissonniere, refactored the storage migration logic, and available for review ...